### ISSUE TITLE
console error on Home page in Firefox

### ISSUE THREAD

**Issue by sujato** [suttacentral/suttacentral#2002]
- Clear all cache and storgae
- load staging Home page in FF
- console error
- 
```
Content Security Policy: The page’s settings blocked the loading of a resource at eval (“script-src”). main.js:784:8441
missing translation key Home 4 main.js:2:75772
Uncaught TypeError: t.path is undefined
    _anchorHandler https://staging.suttacentral.net/main.js:1293
    listen https://staging.suttacentral.net/main.js:1293
    connectedCallback https://staging.suttacentral.net/main.js:1310
    __insert https://staging.suttacentral.net/main.js:2
    __commitNode https://staging.suttacentral.net/main.js:2
    __commitTemplateResult https://staging.suttacentral.net/main.js:2
    commit https://staging.suttacentral.net/main.js:2
    s https://staging.suttacentral.net/main.js:2
    render https://staging.suttacentral.net/main.js:2
    update https://staging.suttacentral.net/main.js:2
    performUpdate https://staging.suttacentral.net/main.js:2
    _enqueueUpdate https://staging.suttacentral.net/main.js:2
    requestUpdateInternal https://staging.suttacentral.net/main.js:2
    set https://staging.suttacentral.net/main.js:2
    __siteLanguageChanged https://staging.suttacentral.net/main.js:2
    _stateChanged https://staging.suttacentral.net/main.js:2
    _stateChanged https://staging.suttacentral.net/main.js:3923
    connectedCallback https://staging.suttacentral.net/main.js:2
    connectedCallback https://staging.suttacentral.net/main.js:3923
    <anonymous> https://staging.suttacentral.net/main.js:3795
    a https://staging.suttacentral.net/main.js:2
    <anonymous> https://staging.suttacentral.net/main.js:2
    <anonymous> https://staging.suttacentral.net/main.js:2
```

**Comment by sujato** [comment]
This error is still current.

**Comment by sujato** [comment]
no longer seeing this.

**Comment by ihongda** [comment]
>Uncaught TypeError: t.path is undefined

This bug has been resolved in PR #2007
